pac2go.sh

```diff
@@ -1,25 +1,25 @@
-#!/usr/bin/env bash
+###############################################################################
+#
+# This script regenerates the source files that embed the pac-cmd executable.
+#
+###############################################################################
 
 function die() {
   echo $*
   exit 1
 }
 
-function gen() {
-  bin=$1
-  dst=$2
-  if [ -f "$bin" ]
-  then
-    echo Generating $dst from $bin...
-    2goarray pacBytes pac < $bin > $dst
-  else
-    echo $bin does not exist, skipping...
-  fi
-}
+if [ -z "$BNS_CERT" ] || [ -z "$BNS_CERT_PASS" ]
```

Comment:
Oh but I don't have access to bns certs, same for other potential users of this utility.

byteexec will ensure file content integrity, even before byteexec was introduced, the binary will be overrode if the permission was altered by coping.